verilog/tools/ls/verilog-language-server.cc

```diff
@@ -138,6 +139,16 @@ void VerilogLanguageServer::SetRequestHandlers() {
         return symbol_table_handler_.FindReferencesLocations(p,
                                                              parsed_buffers_);
       });
+  dispatcher_.AddRequestHandler(
+      "textDocument/prepareRename",
+      [this](const verible::lsp::PrepareRenameParams &p) {
```

Comment:
If we do the optional `lsp::Range` or `null` return here as discussed above, we might need to help out the compiler to deduce the return type by adding `-> nlohmann::json`.